### PR TITLE
Add CreateDepartmentCommandTests

### PR DESCRIPTION
**Pull request by asyagraf**
http://outsourcing.nat.tepkom.ru:20080/redmine/issues/964

### CODE REVIEW

File: test/DepartmentService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    {
+      _autoMocker = new();
+
+      _autoMocker
+        .Setup<IHttpContextAccessor, int>(a => a.HttpContext.Response.StatusCode)
+        .Returns(200);
+
+      _autoMocker
+        .Setup<IResponseCreator, OperationResultResponse<Guid?>>(
+        x => x.CreateFailureResponse<Guid?>(HttpStatusCode.BadRequest, It.IsAny<List<string>>()))
```
Comment (MariyaSin) [inline test/DepartmentService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs:45]: add tab

File: test/DepartmentService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        .Returns(new OperationResultResponse<Guid?>()
+        {
+          Status = OperationResultStatusType.Failed,
+          Errors = new() { "Request is not correct." }
+        });
+
+      _autoMocker
+        .Setup<IResponseCreator, OperationResultResponse<Guid?>>(
+        x => x.CreateFailureResponse<Guid?>(HttpStatusCode.Forbidden, It.IsAny<List<string>>()))
```
Comment (MariyaSin) [inline test/DepartmentService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs:54]: add tab

File: test/DepartmentService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+      _department = new()
+      {
+        Id = Guid.NewGuid(),
+        CompanyId = Guid.NewGuid(),
+        Name = "Name",
+        Description = "Description",
+        IsActive = true,
+        CreatedBy = Guid.NewGuid(),
+        CreatedAtUtc = DateTime.UtcNow,
+        ModifiedBy = Guid.NewGuid(),
```
Comment (MariyaSin) [inline test/DepartmentService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs:79]: null when create department, remove property

File: test/DepartmentService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+        ModifiedAtUtc = DateTime.UtcNow
```
Comment (MariyaSin) [inline test/DepartmentService.Business.UnitTests/Commands/Department/CreateDepartmentCommandTests.cs:80]: same

**Review by MariyaSin — APPROVED** [review]

**Review by grinakate — APPROVED** [review]

**Review by VladSpitsin — APPROVED** [review]

**Pull request merged**